tools/parse_soundfont.py

```diff
@@ -1107,14 +1115,22 @@ def process_percussion_set(self):
                 oot_drum.envelope = drum.envelope_enum
                 oot_drum.pan = max(0, min(64 + round(1.27 * drum.pan), 127))
                 oot_drum.index = index
-                oot_drum.release_index = ctypes.c_int8(find_closest_index(drum.release, release_values)).value
+                oot_drum.release_index = ctypes.c_uint8(find_closest_index(drum.release, release_values)).value # release rates are UNSIGNED integers, not SIGNED integers owl...
```

Comment:
the audio compiling scripts expect a signed integer here, the script accounts for this so the binary value is the same as if it was unsigned when it compiles. don't ask me why its like that as i did not write the audio compiling scripts.